src/cmd-buildextend-live

```diff
@@ -415,6 +415,10 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
         shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
+        for key in list(files_with_karg_embed_areas):
+            if re.search(r"/EFI/.*/grub.cfg", key):
+                files_with_karg_embed_areas['/srv/fcos/tmp/buildpost-live/live/boot/grub/grub.cfg'] = files_with_karg_embed_areas[key]
+                files_with_karg_embed_areas.pop(key, None)
```

Comment:
The key should always exist, so you can use the more canonical

```suggestion
                del files_with_karg_embed_areas[key]
```

form.

Reply:
del was add as suggested

Reply:
This doesn't look resolved.